### ISSUE TITLE
Fav icon

### ISSUE THREAD

**Issue by weeman1337** [flossevents/website#12]
Why? A fav icon for example shows up in the browser title bar, bookmarks etc..

**Comment by weeman1337** [comment]
@dreirik could you please check the fav icon on https://test.floss.events/ ? It should show up in the browser tab and if you bookmark the page.

If you don't see it, please do a reload bypassing the cache (e.g. ctrl + f5).

**Comment by dreirik** [comment]
yeah nice! I would change the tab title description however into "// floss.events" only

another thing: I will upload the final changes I made to the logo tomorrow for further use in everything. However, I get a bit puzzled meanwhile if we should prefer a green star over red. I have the feeling green/dark is very modern and the green also goes very well with the link-color chosen. I made some logo-suggestions regarding color here: https://next.3rik.cc/16/index.php/s/gEG7FrMSMfQLCQy
What is your feeling about it?
